web/src/engine/element-wrappers/src/utils.ts

```diff
@@ -23,16 +23,6 @@ export function nestedInstanceOf(Pelem: Event|EventTarget, className: string): b
     scopedClass = Pelem['defaultView'][className];
   } else if(Pelem['ownerDocument']) {
     scopedClass = (Pelem as Node).ownerDocument.defaultView[className];
-  } else if(Pelem['target']) {
-    var event = Pelem as Event;
-
-    if(this.instanceof(event.target, 'Window')) {
-      scopedClass = event.target[className];
-    } else if(this.instanceof(event.target, 'Document')) {
-      scopedClass = (event.target as Document).defaultView[className];
-    } else if(this.instanceof(event.target, 'HTMLElement')) {
-      scopedClass = (event.target as HTMLElement).ownerDocument.defaultView[className];
-    }
```

Comment:
From https://github.com/keymanapp/keyman/pull/11459#discussion_r1604510440:

> Another case of "there's no reasonable `this` supporting the specified method."  Turns out, we have no cases of calling `nestedInstanceOf` with an event object; we'd have been getting null/undefined errors otherwise.  The code path is actually _dead_ at the moment.
>
> That said, were we to change paths on this fact in the future, the easiest approach is to simply get the event target and act as if that was the original parameter.  Compare with lines 20-25 - they're checking for the same things once we substitute `event.target` in for `Pelem`.